VideoApp/VideoApp/Video/LocalMediaController.m

```diff
@@ -119,13 +121,22 @@ - (void)destroyLocalVideoTrack {
 }
 
 - (void)flipCamera {
-    [self.camera flipCamera];
+    [self.camera flipCamera:self.useMultipartyMedia];
 }
 
 - (BOOL)shouldMirrorLocalVideoView {
     return self.camera.shouldMirrorLocalVideoView;
 }
 
+- (void)checkVideoSenderSettings:(BOOL)isMultiparty {
+    if (SwiftToObjc.isVideoCodecVP8Simulcast && self.localParticipant != nil && isMultiparty != self.useMultipartyMedia) {
+        self.useMultipartyMedia = isMultiparty;
+        TVIEncodingParameters *encodingParameters = [[TVIEncodingParameters alloc] initWithAudioBitrate:0 videoBitrate:1800];
```

Comment:
The is only changing the video bitrate to `kSimulcastVideoBitrate`, which is `1800`.

The code in the Multi-Party example app is doing a few other comparisons:

```
var bitrate = isMultiparty ? CaptureDeviceUtils.kMultipartyVideoBitrate : CaptureDeviceUtils.kOneToOneVideoBitrate
            bitrate = MultiPartyViewController.isSimulcast ? CaptureDeviceUtils.kSimulcastVideoBitrate : bitrate
            localParticipant.setEncodingParameters(EncodingParameters(audioBitrate: MultiPartyViewController.audioBitrate,
                                                                      videoBitrate: bitrate))
```

Where:
```
    static let kOneToOneFrameRate = UInt(24)
    static let kOneToOneVideoBitrate = UInt(1120)
    static let kMultipartyFrameRate = UInt(15)
    static let kMultipartyVideoBitrate = UInt(600)
    static let kSimulcastVideoBitrate = UInt(1800)
    static let kOneToOneVideoDimensions = CMVideoDimensions(width: 640, height: 480)
    static let kMultipartyVideoDimensions = CMVideoDimensions(width: 480, height: 360)
    // Produce 3 spatial layers ~ {1024x768, 512x384, 256x192}
    static let kSimulcastVideoDimensions = CMVideoDimensions(width: 1024, height: 768)
    static let kSimulcastVideoFrameRate = UInt(20)
```

So this only is working in the simulcast case. In the non-simulcast cases, `VP8` or `H.264` it should be able to go between `kOneToOneVideoBitrate` and `kMultipartyVideoBitrate`. In the cases of simulcast its always `kSimulcastVideoBitrate`.

Reply:
Yeah that for sure wasn't right. I think it's fixed now. It's still a little simpler than MultiPartyExample because it only gets inside of this `if` in the case of simulcast.